Bump version to 0.4.9

zanieb opened the pull request.

zanieb commented:
Reverting https://github.com/astral-sh/uv/pull/5927 because it changed a bunch of versions and CI is failing.

charliermarsh commented on .github/workflows/ci.yml:
```diff
@@ -1524,7 +1524,7 @@ jobs:
           cargo run --bin uv -- pip compile scripts/requirements/airflow.in --universal --exclude-newer 2024-08-08 --cache-dir .cache
 
       - name: "Build benchmarks"
-        run: cargo codspeed build --profile profiling --features codspeed -p bench
```
Was this intended to be removed?

zanieb replied:
Yeah, but it's restored now that https://github.com/astral-sh/uv/pull/7276 is merged

zanieb commented:
https://github.com/astral-sh/uv/pull/7276 reverted the lock changes so I've rebased onto `main` and dropped the revert of https://github.com/astral-sh/uv/pull/5927
